homeassistant/components/history_stats/sensor.py

```diff
@@ -344,9 +391,7 @@ def pretty_ratio(value, period):
         """Format the ratio of value / period duration."""
         if len(period) != 2 or period[0] == period[1]:
             return 0.0
-
-        ratio = 100 * 3600 * value / (period[1] - period[0]).total_seconds()
-        return round(ratio, 1)
+        return 100 * 3600 * value / (period[1] - period[0]).total_seconds()
```

Comment:
People will bring pitchforks if they history get values 0.31232312312312312312312

Reply:
The rounding operation for `ratio` type sensors was moved from the static method you mentioned to  the `state` property (L208-212).  That is, the ratio value is rounded immediately prior to returning the state value of the sensor.  By default, it will always round to 2 decimals but with the addition of the new `precision:` configuration variable, users can now modify the number of decimals as well.